### DISCUSSION TITLE
Suggest adding a setting option in Logdy UI to allow users to display the latest logs at the top.

### DISCUSSION THREAD

**Discussion by lwmacct** [logdyhq/logdy-core#46] [Ideas]
Suggest adding a setting option in Logdy UI to allow users to display the latest logs at the top.

**Comment by lwmacct** [comment]
Every time I check the latest log, I have to scroll all the way to the bottom, which is somewhat inconvenient.

**Reply by PeterOsinski** [reply]
Hey thanks, I'll try to come up with something useful

**Reply by PeterOsinski** [reply]
Since v0.14 its possible: https://logdy.dev/blog/post/logdy-new-version-announcement-v014#ordering-log-entries

**Reply by lwmacct** [reply]
> Since v0.14 its possible: https://logdy.dev/blog/post/logdy-new-version-announcement-v014#ordering-log-entries

Thank you for the update! I’ll go ahead and try it now.
